DEFINE.md

```diff
@@ -110,13 +110,13 @@ The most important functions of *param* are:
 
 * *param:getApplicationContext()*: see remarks below
 * *param:getThis()*: the current object instance or *nil* if the method is static
-* *param:getArgument(index)*: get the argument at the specified index (one based)
+* *param:getArgument(index)*: get the argument at the specified index (zero-based)
```

Comment:
This is indeed correct.

Reply:
I'm confused then. I have the latest version of XPrivacyLua on my device and getArgument is zero-based. Passing 1 to it causes an out of bounds error.

Reply:
I meant that your change is correct.